sockcompat.h

```diff
@@ -50,7 +50,9 @@
 #include <ws2tcpip.h>
 #include <stddef.h>
 
+#ifdef _MSC_VER
 typedef signed long ssize_t;
```

Comment:
What about win64?

Reply:
How so? `_MSC_VER` is defined for all versions of the Microsoft Visual C++ compiler (and `_WIN32` is defined for both 32-bit and 64-bit Windows).

Reply:
By the way: Both the new travis builders (MinGW and MSVC) do 64-bit builds.

Reply:
nvm, i forgot that `long` becomes 64 bits on 64 bit platforms.